package.json

```diff
@@ -24,5 +24,9 @@
     "patchedDependencies": {
       "ember-cli-addon-docs@7.0.1": "patches/ember-cli-addon-docs@7.0.1.patch"
     }
+  },
+  "volta": {
+    "node": "18.20.5",
+    "pnpm": "8.15.9"
   }
```

Comment:
I struggled finding the correct PNPM version used in this project. Let me know if I should move it out again.